test/testDAOConfigs.ts

```diff
@@ -0,0 +1,28 @@
+import { DecentDAOConfig } from './../DaoBuilder/types';
+import { ethers } from "ethers";
+
+export const decentTestConfig: DecentDAOConfig = {
+  lockStart: Math.floor(Date.now() / 1000), // (Now) Start time of token lock
+  lockDuration: 60, // 60 seconds
+  snapshotURL: "https://snapshot.org/#/decentdao.eth",
```

Comment:
despite this param being called a URL, we actually only store the ENS name.... 🤦 so ultimately this should just be:

`decent-dao.eth` with a dash